src/components/SignUpForm.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from 'react';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+
+function SignUpFormHeader() {
+  return <h3>Luo uusi käyttäjä</h3>;
+}
+
+function SignUpForm() {
+  const [values, setValues] = useState({
+    firstName: '',
+    lastName: '',
+    userId: '',
+    password: '',
+    passwordConfirm: '',
```

Comment:
Wouldn't `undefined` be clearer as the initialization value?

Reply:
"string is not assignable to a type undefined."
![image](https://user-images.githubusercontent.com/19835914/193458101-96056499-930c-4c28-a608-d0ac538b115a.png)